libtsuba/include/tsuba/tsuba.h

```diff
@@ -97,11 +97,17 @@ struct KATANA_EXPORT RDGView {
   bool transpose{false};
 };
 
-/// list the views in storage for a particular RDG
+/// list the views in storage for a particular version of an RDG
 /// \param rdg_dir is the RDG's URI prefix
 /// \param version is an optional version argument, if omitted this will return
 ///    the views for the latest version
-/// \returns a pair (RDG version, vector of RDGViews)
+/// \returns a pair (RDG version, vector of RDGViews) or ErrorCode::NotFound if
+/// rdg_dir contains no manifest files
+KATANA_EXPORT katana::Result<std::pair<uint64_t, std::vector<RDGView>>>
+ListViewsOfVersion(
+    const std::string& rdg_dir, std::optional<uint64_t> version = std::nullopt);
+
+/// deprecated; duplicate of ListViewsOfVersion maintained for compatibility
 KATANA_EXPORT katana::Result<std::pair<uint64_t, std::vector<RDGView>>>
```

Comment:
How will we know when it is time to get rid of this function?

Reply:
There are some calls in the enterprise repo that need to be switched over

Reply:
Is there a follow-on PR to switch those over?

Reply:
Not yet. I can open one. But in general my sense is that an interface change is best implemented as an interface addition followed by an interface deletion. This way we don't have a tightly-coupled enterprise PR.

Reply:
100% agree with this methodology.  I just worry that it can end up being interface addition followed by, "Squirrels!"